src/testing/testing-sys.ts

```diff
@@ -62,6 +62,8 @@ export const createTestingSystem = (): TestingSystem => {
   sys.writeFile = wrapWrite(sys.writeFile);
   sys.writeFileSync = wrapWrite(sys.writeFileSync);
 
+  sys.getCompilerExecutingPath = () => 'bin/stencil.js';
```

Comment:
tbh I can't remember if it was necessary for code that I ended up including in this PR or not...but yeah while I was trying to get tests together I ran into an error that was addressed by adding this.

Reply:
yeah I must have ended up not including the test code that exercised this, because the tests as I have them pass without it. Do you think we should remove it? It does make the `sys` mock more complete I think.

Reply:
True - while it does make it more complete, my vote would be to remove it for now. Otherwise, we suggest that its a necessary part of this commit, which isn't entirely the case here (I'm thinking about this from a code-archaeology perspective here of "why did they add this when they did?").

Reply:
ok so the plot thickens - I added this line when I was working on getting some of the other test files in `src/compiler/output-targets/test/` working. Then it _wasn't_ being used when I took those changes out of this branch, but then i added [this test](https://github.com/ionic-team/stencil/pull/3368/files#diff-36a88c9676811408967e56ee98189bf643379f9689c83a0ef03f7ad9cd26bfccR116) which does need that line to be present:

```ts
  describe('bundleCustomElements', () => {
    it('should set a diagnostic if no `dir` prop on the output target', async () => {
      const { config, compilerCtx, buildCtx } = setup();
      const outputTarget: OutputTargetDistCustomElements = { type: 'dist-custom-elements' };
      await bundleCustomElements(config, compilerCtx, buildCtx, outputTarget);
      expect(buildCtx.diagnostics).toEqual([
        {
          level: 'error',
          type: 'build',
          messageText: 'dist-custom-elements output target provided with no output target directory!',
        },
      ]);
    });
  });
```

so I think we do need to include it after all. I could add a comment point to our conversation here for future conversation if necessary to clarify why it's in this commit in particular

Reply:
gotcha - no comment needed then 👍